### PR TITLE
Cpython pinning

### PR DESCRIPTION
**Pull request by isuruf**
Blocked by https://github.com/conda-forge/conda-smithy/pull/1244

### CODE REVIEW

**Comment by jakirkham** [comment]
How does this relate to PR ( https://github.com/conda-forge/conda-smithy/pull/1244 )?

**Comment by isuruf** [comment]
conda-smithy messes up the rerender with the new `conda_build_config.yaml` from this PR and the `python38.yaml` migration file from the feedstocks, which has old version. See my two comments at https://github.com/conda-forge/conda-smithy/issues/1243

**Comment by beckermr** [comment]
Man this linter failure is not great. Do we need quotes around the ** in the strings?

**Comment by isuruf** [comment]
Hmm, I can't reproduce locally? What's the error message?

**Comment by beckermr** [comment]
@conda-forge-admin relint

**Comment by beckermr** [comment]
yay!

**Comment by isuruf** [comment]
Thanks @beckermr. This is ready for review.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,8 +11,12 @@ build:
...
-    - copy conda_build_config.yaml %PREFIX%   # [win]
+    - cp conda_build_config.yaml $PREFIX                       # [unix]
+    - mkdir -p $PREFIX/share/conda-forge/migrations            # [unix]
+    - cp migrations/* $PREFIX/share/conda-forge/migrations/    # [unix]
+    - rm $PREFIX/share/conda-forge/migrations/example.exyaml   # [unix]
+    - echo "This package can't be built on windows"            # [win]
+    - exit 1                                                   # [win]
 
 test:
   commands:
```
Comment (beckermr) [inline recipe/meta.yaml]: Do you want to test for the migrations in the right spot @isuruf ?

**Comment by isuruf** [comment]
Thanks for the review

**Pull request merged**